src/module.js

```diff
@@ -118,8 +118,10 @@ const FRAME_PROPS = {
   },
   lang: { 
     validate: (lang) => {
-      return lang === 'en-us' || lang === 'en' || lang === 'fr';
-    }
+      return lang === 'de' || lang === 'en-us' || lang === 'en' || lang === 'fi' ||
```

Comment:
might be easier to maintain following the pattern `return ['de','en-us','en','fi'].includes(lang);`

Reply:
oh that's nice. i'll do that. i forget about `includes`

Reply:
updated to your suggestion and in the process found out that 'user' is only an option at the domain/room rest api level. Not from the front-end... so I've removed that.